dist/index.html

```diff
@@ -11,63 +11,62 @@
-      />
-      <img
-        src="./logo_transparent.png"
+        src="./images/logo_transparent.png"
         alt="fitlit logo"
-        height="100px"
-        width="100px"
+        id="fitlitLogo"
+        height="300px"
+        width="300px"
       />
     </header>
-    <main class="vitals-section-main">
-      <section class="sleep-container">
-        <img
-          src="./lighting.png"
-          alt="sunset logo"
+    <main>
+      <section class="left-nav">
+        <section class="vitals-section">
+          <section class="sleep-container">
+            <img
+            src="./images/lighting.png"
+            id="sleepImg"
+            alt="sunset logo"
+            height="50px"
+            width="50px"/>
+            <h2 class="sleep-header">SLEEP</h2>
+            <section class="sleep-card">sleep goes here
+            </section>
+          </section>
+          <section class="hydration-container">
+            <img
+              src="./images/water-bottle.png"
+              id="waterBottleImg"
+              alt="water bottle"
+              height="50px"
+              width="50px"/>
+            <h2 class="hydration-header">HYDRATION</h2>
+            <section class="hydration-card">hydration goes here
+            </section>
+          </section>
+        </section> 
+        <section class="steps-container">
+          <img
+          src="./images/sandals.png"
+          id="sandalsImg"
+          alt="weight logo"
           height="50px"
           width="50px"
-        />
-        <h2 class="sleep-header">SLEEP</h2>
-        <section class="sleep-card">
+          />
+          <h2 class="steps-header">ACTIVITY</h2>
+          <section class="step-card"></section>
         </section>
       </section>
-      <section class="hydration-container">
-        <img
-          src="./water-bottle.png"
-          alt="water bottle"
-          height="50px"
-          width="50px"
-        />
-        <h2 class="hydration-header">HYDRATION</h2>
-        <section class="hydration-card">
+      <aside class="right-nav">
+        <section class="user-container">
+          <section class="user-card">
+          </section>
         </section>
-      </section>
-    </main>
-    <aside class="right-nav">
-      <section class="user-container">
-        <img
-          src="./avatar-male.png"
-          alt="male avatar"
-          height="50px"
-          width="50px"
-        />
-        <section class="user-card">
-        </section>
-      </section>
-      <section class="friends-container">
-        <h2 class="friends-header">Friends</h2>
-        <section class="friends-card"></section>
-      </section>
-      <section class="step-comparison-container">
-        <h2 class="steps-header">How Your Steps Measure Up</h2>
-        <section class="step-card">
+        <section class="friends-container">
+          <h2 class="friends-header">Friends</h2>
+          <section class="friends-card"></section>
```

Comment:
I add `<ul> </ul>` tags I made a PR I hope you see it.

Reply:
I don't think those are actually needed in this case since I don't think bullets are needed for styling.

Reply:
as I said I can explain later over zoom